<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_login</class>
 <widget class="QWidget" name="Form_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOGIN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources_rc/resources.qrc">
    <normaloff>:/LogoAndWindowsIcon/Aset/LogoIconSolusiku-final.png</normaloff>:/LogoAndWindowsIcon/Aset/LogoIconSolusiku-final.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ebebea;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_usernames">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid transparent;
border-radius : 9px;
background-color: white;
color:#666666;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Masukkan Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_passwords">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border : 1px solid transparent;
border-radius : 9px;
background-color: white;
color:#666666;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Masukkan Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Login">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #5ca3ff;
border-radius: 9px;
background-color: #5ca3ff;
color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_Regist">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>261</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
border-radius: 9px;
background-color: #ffffff;
color: gray;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : transparent;
</string>
   </property>
   <property name="text">
    <string>_________________</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>32</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string>or</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>-40</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources_rc/resources.qrc">:/Images/Aset/Logo-removebg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>892</width>
     <height>425</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources_rc/resources.qrc">:/Images/Aset/Shadow_blue.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #666666;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string> Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #666666;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string> Password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_show_Pw">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>308</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: #666666;</string>
   </property>
   <property name="text">
    <string>Show Password</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : transparent;
</string>
   </property>
   <property name="text">
    <string>________________</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetpw">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: #2d48b8;</string>
   </property>
   <property name="text">
    <string>Reset password?</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>txt_usernames</zorder>
  <zorder>txt_passwords</zorder>
  <zorder>btn_to_Regist</zorder>
  <zorder>btn_Login</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>chk_show_Pw</zorder>
  <zorder>label_9</zorder>
  <zorder>btnResetpw</zorder>
 </widget>
 <resources>
  <include location="../Resources_rc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
